src/pe_reports/report_generator.py

```diff
@@ -55,6 +62,18 @@ def load_template():
     return prs
 
 
+def load_customers():
+    """Export PowerPoint report set to output directory."""
+    try:
+
+        if os.path.getsize(CUSTOMERS) != 0 and os.path.exists(CUSTOMERS):
+            with open(CUSTOMERS) as customers_file:
+                return json.load(customers_file)
+    except FileNotFoundError as not_found:
+        logging.error("%s : Missing input data. No report generated.", not_found)
+        return
```

Comment:
This keeps the return type consistent.

```suggestion
        return dict()
```